### PR TITLE
set max sell value on token selection (swaps)

### PR DESCRIPTION
**Pull request by magiziz**
Fixes BX-1566
Figma link (if any):

## What changed (plus any additional context for devs)

Once sell token has been selected we set max input value.

## Screen recordings / screenshots

https://github.com/user-attachments/assets/d2b797a4-bf55-45bb-b016-81e6762ac3d9

## What to test

Go to swaps, select a token and make sure the input value is set to max.

### CODE REVIEW

**Review by estebanmino — APPROVED** [review]

**Review by DanielSinclair — APPROVED** [review]
lgtm

File: src/entries/popup/pages/swap/index.tsx [hunk 1]
```diff
@@ -454,8 +456,12 @@ export function Swap({ bridge = false }: { bridge?: boolean }) {
       if (!assetToBuy) tokenToBuyInputRef.current?.openDropdown();
       setAssetToSellInputValue('');
       setAssetToBuyInputValue('');
+      if (asset) {
+        setHasRequestedMaxValueAssetToSell(true);
```
Comment (greg-schrammel) [inline src/entries/popup/pages/swap/index.tsx:460]: instead of a flag that triggers the `useEffect` can't we just put `setAssetToSellMaxValue();` here?

Comment (greg-schrammel) [reply src/entries/popup/pages/swap/index.tsx:460]: ah I see 🫠 hate it but seems like only refactoring everything to do that

**Review by greg-schrammel — APPROVED** [review]

**Comment by magiziz** [comment]
Chrome swap e2e test seems to fail, but works locally. Going to see what's going on.

image

**Pull request merged**